packages/jest-resolve-dependencies/src/__tests__/dependency_resolver.test.ts

```diff
@@ -31,7 +31,11 @@ beforeEach(async () => {
     cacheDirectory: path.resolve(tmpdir(), 'jest-resolve-dependencies-test'),
     moduleDirectories: ['node_modules'],
     moduleNameMapper: [['^\\$asdf/(.*)$', '<rootDir>/$1']],
-    rootDir: '.',
+    /**
+     * `filter` must be an absolute path
```

Comment:
```suggestion
     * `rootDir` must be an absolute path
```

Ops, a typo, I'll change it in another PR as a tiny side effect.